src/PKSim.Presentation/Presenters/PopulationAnalyses/PopulationAnalysisChartPresenter.cs

```diff
@@ -101,6 +112,15 @@ public virtual void ClearPlot()
          _chartDataBinder.ClearPlot();
       }
 
+      public void ExportDataToImage()
+      {
+         var fileName = _dialogCreator.AskForFileToSave(PKSimConstants.UI.ExportPopulationAnalysisToExcelTitle, Constants.Filter.DIAGRAM_IMAGE_FILTER, Constants.DirectoryKey.REPORT, AnalysisChart.Name);
```

Comment:
`PKSimConstants.UI.ExportPopulationAnalysisToExcelTitle` should be replaced by `...ExportToPNG` I guess

Reply:
Indeed. Fixed.